### PR TITLE
2022.2

### PR DESCRIPTION
**Pull request by chrmod**
TODO:
- [x] Sentry integration
- [x] GBE update

### CODE REVIEW

File: patches/0012-Disabling-Mozilla-Tracking-protection.patch [hunk 1]
```diff
@@ -1,64 +1,64 @@
-From f0137dcd9fedc47784009553efbbeb90734ef42d Mon Sep 17 00:00:00 2001
+From 802411ca0de5a3564f465d515466ff8f4ed3d9f6 Mon Sep 17 00:00:00 2001
```
Comment (philipp-classen) [inline patches/0012-Disabling-Mozilla-Tracking-protection.patch:1]: Would it make sense to keep Mozilla's tracking protection enabled? Might mess up the visualization and antitracking/WTM related stats though.

**Review by philipp-classen — APPROVED** [review]

File: patches/0012-Disabling-Mozilla-Tracking-protection.patch (hunk 1 above)
Comment (chrmod) [reply patches/0012-Disabling-Mozilla-Tracking-protection.patch:1]: AFAICT Ghostery provides stronger protection. No need to run both.

**Review by DamarcusH1 — COMMENTED** [review]
😎

**Comment by DamarcusH1** [comment]
😎

**Pull request merged**